update example configuration of kafka sink in user guide

Robin021 opened the pull request:
line 2833 change ki to k1

Robin021 committed:
update example configuration of kafka sink 

line 2833 change ki to k1

bessbd reviewed (approved):
Looks good to me, +1

Robin021 committed:
Customization of netcat source new line identifier

Some special request from customer , they do not want use \n as the new line, so add some configurable code for this feature. change made in NetcatSourceConfigurationConstants.java and NetcatSource.java
CONFIG_END_LINE_IDENTIFIER = "end-line-identifier"; 
DEFAULT_CONFIG_END_LINE_IDENTIFIER = 0x0A ;

adenes commented:
Hi @Robin021 ,
Thank you for your patch.
The fist commit looks good to me as well, but it seems that you accidentally pushed a new change to this branch/pull request.
If you are fine with it we can push the [first commit](https://github.com/apache/flume/pull/165/commits/83e5719da6be52a86d6318f87d396ef64d294648) to the main branch and close this PR.

Robin021 commented:
Yes, please commit the first commit.  
other changes are only customer use of my testing program , will update later. 

thanks!
